modules/channel-vonage/src/actions/storeFileLocally.js

```diff
@@ -0,0 +1,49 @@
+const axios = require('axios')
+const FormData = require('form-data')
+const uuidv4 = require('uuid').v4
+
+/**
+ * Store File Locally
+ * @title Store File Locally
+ * @category Channel Vonage
+ * @author Botpress, Inc.
+ * @param {string} file - The URL of the file to store locally
+ */
+const storeFileLocally = async file => {
+  if (!event.payload.url || !file) {
+    return
+  }
+
+  try {
+    const resp = await axios.get(file, { responseType: 'arraybuffer' })
+
+    const filename =
+      (resp.headers['content-disposition'] &&
+        resp.headers['content-disposition'].split('filename=')[1].replace(/"/g, '')) ||
```

Comment:
What does these two lines do? Maybe put a comment to clarify

Reply:
Good point I put it into a separate function with comments explaining what it does.